include/zephyr/drivers/lora.h

```diff
@@ -46,13 +46,36 @@ enum lora_coding_rate {
 };
 
 struct lora_modem_config {
+	/* Frequency in Hz to use for transceiving */
```

Comment:
Nitpick: Zephyr prefers each commit to do a single thing.
Adding documentation for the existing parameters and adding the new ones should be 2 seperate commits.